README.md

```diff
@@ -16,65 +16,68 @@
 [travis]: https://travis-ci.org/mcu-tools/mcuboot
 [license]: https://github.com/mcu-tools/mcuboot/blob/main/LICENSE
 
-This is mcuboot version 1.8.0-rc1
+This is MCUboot version 1.8.0-rc1
 
 MCUboot is a secure bootloader for 32-bit MCUs. The goal of MCUboot is to
 define a common infrastructure for the bootloader, system flash layout on
 microcontroller systems, and to provide a secure bootloader that enables
 simple software upgrades.
 
-MCUboot is operating system and hardware independent and relies on
-hardware porting layers from the operating. Currently, mcuboot works
-with both the Apache Mynewt and Zephyr operating systems, but more
-ports are planned in the future. RIOT is currently supported as a boot
-target with a complete port planned.
+MCUboot is operating-system and hardware independent and relies on
+hardware porting layers from the operating system. Currently, MCUboot
+works with the following operating systems:
+- [Zephyr](https://www.zephyrproject.org/)
+- [Apache Mynewt](https://mynewt.apache.org/)
+- [Apache NuttX](https://nuttx.apache.org/)
+- [RIOT](https://www.riot-os.org/)
+- [Mbed OS](https://os.mbed.com/)
+
+RIOT is currently supported only as a boot target. There is currently no
+defined plan to add any additional port. However, we will accept any new
+port contributed by the community once it is good enough.
 
 ## Using MCUboot
 
-Instructions for different operating systems can be found here:
+Instructions for different operating systems can be found below:
 - [Zephyr](docs/readme-zephyr.md)
-- [Mynewt](docs/readme-mynewt.md)
+- [Apache Mynewt](docs/readme-mynewt.md)
+- [Apache NuttX](docs/readme-nuttx.md)
 - [RIOT](docs/readme-riot.md)
-- [Mbed-OS](docs/readme-mbed.md)
-- [NuttX](docs/readme-nuttx.md)
-- [Espressif IDF](docs/readme-espressif.md)
-- [Simulator](sim/README.rst)
+- [Mbed OS](docs/readme-mbed.md)
+
+There are also instructions for the [Espressif
+IDF](docs/readme-espressif.md) SDK and for the
+[Simulator](sim/README.rst).
 
 ## Roadmap
 
 The issues being planned and worked on are tracked using GitHub issues. To
-participate please visit:
-
-[MCUBoot GitHub Issues](https://github.com/mcu-tools/mcuboot/issues)
-
-~~Issues were previously tracked on [MCUboot JIRA](https://runtimeco.atlassian.net/projects/MCUB/summary)
-, but it is now deprecated.~~
+give your input, visit [MCUBoot GitHub
+Issues](https://github.com/mcu-tools/mcuboot/issues).
 
 ## Browsing
 
-Information and documentation on the bootloader are stored within the source.
-
-~~It was previously also documented on confluence:
-[MCUBoot Confluence](https://runtimeco.atlassian.net/wiki/discover/all-updates)
-however, it is now deprecated and not currently maintained~~
+Information and documentation on the bootloader are stored within the
+source.
 
-For more information in the source, here are some pointers:
+For more information, use the following pointers to the source:
```

Comment:
No comma before "use": the "For more information" defines what the pointers are for, so it is restrictive.
Shouldn't we use "links" for "on-line" documentation rather than "pointers" ?
In line 79 we use "links".

Reply:
Right, rephrased it completely.